test/e2e/xsite/xsite_test.go

```diff
@@ -231,6 +231,16 @@ func TestDefaultTLSLoadBalancerWithPort(t *testing.T) {
 	testCrossSiteView(t, true, ispnv1.CrossSiteSchemeTypeOpenShift, ispnv1.CrossSiteExposeTypeLoadBalancer, 1443, 1, DefaultTLS, nil)
 }
 
+func TestCrossSiteViewOpenshiftRoute(t *testing.T) {
+	tutils.SkipForExposeType(t, ispnv1.ExposeTypeNodePort, ispnv1.ExposeTypeLoadBalancer)
+	testCrossSiteView(t, true, ispnv1.CrossSiteSchemeTypeOpenShift, ispnv1.CrossSiteExposeTypeRoute, 0, 1, NoTLS, nil)
```

Comment:
`NoTLS` option is confusing. Cross site over Route won't work without TLS

Reply:
I'm going to remove this test. It does not make sense to me :rofl: